roles/opensrp/tasks/main.yml

```diff
@@ -74,3 +74,12 @@
   service:
     name: "{{ opensrp_tomcat_instance }}"
     state: restarted
+
+- name: Pause 15 secs to allow tomcat to extract war
+  pause:
```

Comment:
Can we use [`wait_for`](https://docs.ansible.com/ansible/latest/modules/wait_for_module.html) here?

Reply:
nice, wait_for looks better. Let me update to wait for a specific path to be available